### ISSUE TITLE
Permissions

### ISSUE THREAD

**Issue by IzzySoft** [ggsava/block-this#22]
Just curious: for what does an AdBlocker need the [READ_PHONE_STATE](https://android.izzysoft.de/applists/perms#READ_PHONE_STATE) permission? This makes the app suspicious to especially privacy-aware people (I got that response when recommending your app).

If it's for the "usual purpose" (i.e. to "background" on incoming calls, which the "official developer documentation" mentions it for): I don't see how that should make sense. An AdBlocker should work in the background anyway, and configuration/settings pages make no sound and play no video. So there's nothing to interfere with phone calls. And even if there were, there are [other solutions available](http://android.stackexchange.com/a/62968/16575).

TL;DR: Would you consider removing that permission requirement?

**Comment by ggsava** [comment]
Thanks @IzzySoft. I thought I already removed that, I will take a look at it again. I think it might be from something I tested in a previous version.. but I believe its not necessary anymore.

**Comment by IzzySoft** [comment]
Thanks for your fast reaction, @ggsava – I will report that back to the reporting user, uh... "on the double" :laughing: Looking forward then to the next version without that permission :+1:

**Comment by IzzySoft** [comment]
Almost a year later: Will there be a release anytime soon to remove that? Last release was quite a while ago, and according to the repo there has been some more development since (though the last change was 3 months back).
